### PR TITLE
Hoogle 5 support

### PR DESCRIPTION
**Pull request by tmcgilchrist**

### CODE REVIEW

File: main/mafia.hs [hunk 1]
```diff
@@ -750,7 +750,7 @@ appendStandardDirs dir =
 
 standardSourceDirs :: [Directory]
 standardSourceDirs =
-  ["src", "test", "gen", "dist/build", "dist/build/autogen"]
+  ["src", "test", "gen", "dist/build", "dist/build/autogen", "dist/build/global-autogen"]
```
Comment (erikd) [inline main/mafia.hs:753]: Is that something new in cabal 2.0 ?

File: src/Mafia/Git.hs [hunk 2]
```diff
@@ -91,7 +91,7 @@ getSubmodules = do
   root    <- getProjectRoot
   Out out <- callFrom GitProcessError root "git" ["submodule"]
 
-  sequence . fmap parseSubmoduleLine . T.lines $ out
+  traverse parseSubmoduleLine . T.lines $ out
```
Comment (erikd) [inline src/Mafia/Git.hs:94]: :)

File: main/mafia.hs (hunk 1 above)
Comment (tmcgilchrist) [reply main/mafia.hs:753]: New in cabal 2.0 and GHC 8.2 apparently.

**Review by erikd — APPROVED** [review]
When CI is happy with this then so am I.

**Comment by tmcgilchrist** [comment]
The CI hooks for travis look like they are still borked, see https://travis-ci.org/haskell-mafia/mafia/builds/444837993

**Review by domdere — COMMENTED** [review]
LGTM, 👍 on 🤢

**Pull request merged**